iphone/TitaniumKit/TitaniumKit/Sources/API/TiBase.h

```diff
@@ -330,7 +330,7 @@ void TiExceptionThrowWithNameAndReason(NSString *exceptionName, NSString *reason
     return [NSNumber numberWithInt:map]; \
   }
 
-#define MAKE_SYSTEM_UINTEGER(name, map)              \
+#define MAKE_SYSTEM_PROP_UINTEGER(name, map)         \
```

Comment:
Please do not rename but add a new one and deprecate the old one. Otherwise, a good hand full of native modules will be broken in SDK 8+.